### PR TITLE
Bump dependencies for Robolectric 4.7.1

### PR DESCRIPTION
**Pull request by utzcoz**
See https://github.com/robolectric/robolectric/pull/6867 and https://github.com/robolectric/robolectric/releases/tag/robolectric-4.7.2.

### CODE REVIEW

**Comment by utzcoz** [comment]
Looks robolectric-bazel doesn't have CI to test new changes. Hi @Bencodes , do you help to review and double-checking it? Thanks.
cc @hoisie @brettchabot .

**Comment by Bencodes** [comment]
@utzcoz want to give https://github.com/robolectric/robolectric-bazel/pull/24 and https://github.com/robolectric/robolectric-bazel/pull/25 a review (will rebase the second one) to address the CI issues?

**Comment by utzcoz** [comment]
> @utzcoz want to give https://github.com/robolectric/robolectric-bazel/pull/24 and https://github.com/robolectric/robolectric-bazel/pull/25 a review (will rebase the second one) to address the CI issues?

Thanks. We can update this project with your PRs more quickly and easily. I will rebase this OR after your two PRs merged.

**Comment by utzcoz** [comment]
Rebased and CI passed.

**Review by hoisie — APPROVED** [review]

**Review by Bencodes — APPROVED** [review]

**Comment by utzcoz** [comment]
@Bencodes maybe I need you help to merge it and do a 4.7.1 release if there is other problems. Thanks.

File: README.md [hunk 1]
```diff
@@ -10,8 +10,8 @@ Add the `robolectric` and `rules_jvm_external` repositories in your WORKSPACE fi
 ```python
 http_archive(
     name = "robolectric",
-    urls = ["https://github.com/robolectric/robolectric-bazel/archive/4.7.tar.gz"],
-    strip_prefix = "robolectric-bazel-4.7",
+    urls = ["https://github.com/robolectric/robolectric-bazel/archive/4.7.1.tar.gz"],
+    strip_prefix = "robolectric-bazel-4.7.1",
```
Comment (Bencodes) [inline README.md:14]: Did you mean to point this at 4.7.2?

Comment (utzcoz) [reply README.md:14]: Thanks for pointing out it. The version of robolectric-bazel should keep up with Robolectric. This release is for Robolectric 4.7.1. After it released, we can update version to 4.7.2 on document only and do another release for Robolectric 4.7.2 with version 4.7.2. @brettchabot  I hope I don't misunderstand version rules of robolectric-bazel, we discussed at other PRs before.

**Comment by utzcoz** [comment]
@Bencodes Thanks.

**Pull request merged**